src/lib/components/inbox/conversations/conversation_snippet.tsx

```diff
@@ -109,7 +111,7 @@ export class ConversationSnippet extends React.Component<Props, any> {
     const artworkDate = `${artwork.date}`
     const artworkArtist = `${artwork.artist.name} · `
     const conversationText = conversation.last_message.replace(/\n/g, " ")
-    const date = "11:00 AM"
+    const date = moment(conversation.last_message_at).fromNow(true)
```

Comment:
from the example data below, I assume that this can return things like "a few seconds"? 

From what I remember, though not scoped [in this ticket](https://github.com/artsy/collector-experience/issues/295), I thought that anything less than a day will say the time, and anything more than a day will say "x days ago". 

Whatcha think?

Happy to ask for clarification and taking it on as a separate item, and getting this in as is :)